README.md

```diff
@@ -1,133 +1,160 @@
 OpenMC Plot Explorer
+====================
 
-Description:
+Description
+-----------
 
 A graphical interface for visualizing and manipulating OpenMC plot slices.
 
-Dependencies:
+Dependencies
+------------
 
-os, sys, pickle, copy, struct, threading, numpy, ast, PySide2
-openmc (plot-ids branch from Paul Romano)
+os, sys, pickle, copy, struct, threading, numpy, ast, PySide2, openmc
```

Comment:
I wouldn't really call standard library modules "dependencies", since they are always present. The only real dependencies here are numpy, PySide2, and openmc.

Reply:
Should add matplotlib too

Reply:
Good call. On it!